### PR TITLE
Fix error division by zero when running on empty folder.

### PR DESCRIPTION
**Pull request by milo-hyben**
Hi Terry, 
in case when lizard does not find any function in the file or folder it crashes. 
I added a simple condition to check for such scenarios.

Cheers,
Milo

### CODE REVIEW

File: lizard_languages/clike.py [hunk 1]
```diff
@@ -135,7 +135,7 @@ def _state_global(self, token):
         if token == "template":
             self._state = self._template_declaration
 
-        elif token in ("struct", "class", "namespace"):
+        elif self.last_token != "." and token in ("struct", "class", "namespace"):
             self._state = self._read_namespace
```
Comment (rakhimov) [inline lizard_languages/clike.py]: Can elaborate on this?
Some examples and tests?

Comment (rakhimov) [inline lizard_languages/clike.py]: This seems to be a separate issue from the fixing of division by zero.
I would suggest to put it into another PR.

Comment (milo-hyben) [inline lizard_languages/clike.py]: Sorry for including that one.
I had reverted back that commit.
It is different issue I will be pushing later, related to Java Class Literals (".class") causing out of memory issue with deep nesting.

**Comment by terryyin** [comment]
@milo-hyben thank you so much for the pull-request. Sorry for the late reply, being travelling for two weeks in a row.

Would you be interested in adding a unit test for this scenario? Or you can tell me one scenario that can reproduce the problem and I will add the test.

**Comment by milo-hyben** [comment]
Hi Terry,
added s simple test for xml output which was causing lizard to crash.
Let me know if you think that is good enough.

**Comment by terryyin** [comment]
Great. Thanks:-)
